.gitignore

```diff
@@ -10,3 +10,4 @@
 /dist/gh-pages/
 /tmp-test-bundle.js
 /coverage/
+.idea
```

Comment:
I don't think this needs to be in the commit

Reply:
it is a separate commit though, 
I could drop it eventually...

Reply:
- [ ] drop .gitignore updates